### ISSUE TITLE
Cannot initialize innodb storage engine with non-default images.

### ISSUE THREAD

**Issue by bluven** [presslabs/mysql-operator#230]
I tried to create mysql cluster with non-default percona images( percona:5.7.24 and percona:5.7-centos), but wierd things happened: the first node started successfully, but the second node had an initialization problem:
```
2019-02-13T07:38:20.744819Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
2019-02-13T07:38:20.751017Z 0 [Note] mysqld (mysqld 5.7.24-26-log) starting as process 1 ...
2019-02-13T07:38:20.752921Z 0 [Warning] Although a path was specified for the --slow-query-log-file option, log tables are used. To enable logging to files use the --log-output=file option.
2019-02-13T07:38:20.774750Z 0 [Note] InnoDB: PUNCH HOLE support available
2019-02-13T07:38:20.774798Z 0 [Note] InnoDB: Mutexes and rw_locks use GCC atomic builtins
2019-02-13T07:38:20.774805Z 0 [Note] InnoDB: Uses event mutexes
2019-02-13T07:38:20.774808Z 0 [Note] InnoDB: GCC builtin __atomic_thread_fence() is used for memory barrier
2019-02-13T07:38:20.774812Z 0 [Note] InnoDB: Compressed tables use zlib 1.2.7
2019-02-13T07:38:20.774816Z 0 [Note] InnoDB: Using Linux native AIO
2019-02-13T07:38:20.775800Z 0 [Note] InnoDB: Number of pools: 1
2019-02-13T07:38:20.776044Z 0 [Note] InnoDB: Using CPU crc32 instructions
2019-02-13T07:38:20.867034Z 0 [Note] InnoDB: Initializing buffer pool, total size = 128M, instances = 1, chunk size = 128M
2019-02-13T07:38:20.882885Z 0 [Note] InnoDB: Completed initialization of buffer pool
2019-02-13T07:38:20.888984Z 0 [Note] InnoDB: If the mysqld execution user is authorized, page cleaner thread priority can be changed. See the man page of setpriority().
2019-02-13T07:38:20.902872Z 0 [ERROR] InnoDB: The innodb_system data file 'ibdata1' must be writable
2019-02-13T07:38:20.902902Z 0 [ERROR] InnoDB: The innodb_system data file 'ibdata1' must be writable
2019-02-13T07:38:20.902912Z 0 [ERROR] InnoDB: Plugin initialization aborted with error Generic error
2019-02-13T07:38:22.007289Z 0 [ERROR] Plugin 'InnoDB' init function returned error.
2019-02-13T07:38:22.007336Z 0 [ERROR] Plugin 'InnoDB' registration as a STORAGE ENGINE failed.
2019-02-13T07:38:22.007347Z 0 [ERROR] Failed to initialize builtin plugins.
2019-02-13T07:38:22.007356Z 0 [ERROR] Aborting

2019-02-13T07:38:22.007376Z 0 [Note] Binlog end
2019-02-13T07:38:22.011461Z 0 [Note] mysqld: Shutdown complete
```

**Comment by AMecea** [comment]
Hi @bluven, the operator works only with images from Percona based on Debian, those based on CentOS, the new ones, does not work because the data is mounted under a different user. So this is a duplicate of #200 and I've already started working at it in PR #224, which will be merged in `v0.2.5`.

**Comment by AMecea** [comment]
this issue is fixed by #224, now you can use any images from Percona for MySQL

**Comment by AMecea** [comment]
in `v0.2.5` to use Percona CentOS images just specify the `mysqlVersion` field on `MysqlCluster` resource to `5.7-centos`, for example:

```yaml
spec:
  replicas: ...
  mysqlVersion: 5.7-centos
```

By default, the operator will use Debian based images but in version `v0.3.x` latest images will be used.
